Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-programming/questions-programming.component.ts

```diff
@@ -26,7 +28,10 @@ export class QuestionsProgrammingComponent implements OnInit {
     isCategoryReady: boolean;
     isLanguageReady: boolean;
     isFormSubmitted: boolean;
+    isTestCaseAdded: boolean;
     code: any;
+    testCases: CodeSnippetQuestionsTestCases[];
+    TestCaseType: TestCaseType;
```

Comment:
it is deliberately kept to make it understood that it enables the enum of TestCaseType in the component.